archinstall/default_profiles/desktops/river.py

```diff
@@ -0,0 +1,78 @@
+from typing import TYPE_CHECKING, override
+
+from archinstall.default_profiles.desktops import SeatAccess
+from archinstall.default_profiles.profile import GreeterType, ProfileType, SelectResult
+from archinstall.default_profiles.xorg import XorgProfile
+from archinstall.tui.curses_menu import SelectMenu
+from archinstall.tui.menu_item import MenuItem, MenuItemGroup
+from archinstall.tui.types import Alignment, FrameProperties, ResultType
+
+if TYPE_CHECKING:
+	from collections.abc import Callable
+
+	from archinstall.lib.translationhandler import DeferredTranslation
+
+	_: Callable[[str], DeferredTranslation]
+
+
+class riverProfile(XorgProfile):
+	def __init__(self) -> None:
+		super().__init__(
+			'river'
+			ProfileType.WindowMgr,
+			description=''
+		)
+
+		self.custom_settings = {'seat_access': None}
+
+	@property
+	@override
+	def packages(self) -> list[str]:
+		additional = []
+		if seat := self.custom_settings.get('seat_access', None):
+			additional = [seat]
+
+		return [
+			"foot",
+			"xdg-desktop-portal-wlr"
+		] + additional
+
+	@property
+	@override
+	def default_greeter_type(self) -> GreeterType | None:
+		return GreeterType.Lightdm
+
+	@property
+	@override
+	def services(self) -> list[str]:
+		if pref := self.custom_settings.get('seat_access', None):
+			return [pref]
+		return []
+
+	def _ask_seat_access(self) -> None:
```

Comment:
I can't see this being mentioned in the wiki https://wiki.archlinux.org/title/River

Reply:
seems to be needed https://libera.irclog.whitequark.org/river/search?q=seat

Reply:
Have you actually tested these profiles?